examples/using-javascript-transforms/src/html.js

```diff
@@ -2,6 +2,15 @@ import React from 'react';
 import * as PropTypes from "prop-types"
 import Helmet from 'react-helmet';
 
+let stylesStr
```

Comment:
How about you just remove this file altogether and use the default html.js? It already has inlining of css setup, etc.